cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -523,6 +532,17 @@ function(px4_os_add_flags)
 
 	set(added_exe_linker_flags) # none currently
 
+
```

Comment:
@dagar I did not see this section [here](https://github.com/PX4/Firmware/blob/master/cmake/nuttx/px4_impl_nuttx.cmake#L441) on master from your hand merge